test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,38 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // No arguments (implied no null value)
+        assertNotHasNull();
+
+        // Singular arguments
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling. Leaving a blank line at line 59 would be enough.